apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatListView.kt

```diff
@@ -298,15 +313,120 @@ fun connectIfOpenedViaUri(rhId: Long?, uri: URI, chatModel: ChatModel) {
     chatModel.appOpenUrl.value = rhId to uri
   } else {
     withApi {
-      planAndConnect(chatModel, rhId, uri, incognito = null, close = null)
+      planAndConnect(rhId, uri, incognito = null, close = null)
     }
   }
 }
 
+@Composable
+private fun ChatListSearchBar(listState: LazyListState, searchText: MutableState<TextFieldValue>, searchShowingSimplexLink: MutableState<Boolean>, searchChatFilteredBySimplexLink: MutableState<String?>) {
+  Row(verticalAlignment = Alignment.CenterVertically) {
+    val focusRequester = remember { FocusRequester() }
+    var focused by remember { mutableStateOf(false) }
+    Icon(painterResource(MR.images.ic_search), null, Modifier.padding(horizontal = DEFAULT_PADDING_HALF), tint = MaterialTheme.colors.secondary)
+    SearchTextField(
+      Modifier.weight(1f).onFocusChanged { focused = it.hasFocus }.focusRequester(focusRequester),
```

Comment:
closing keyboard doesn't remove cursor from search, considering search can't even be hidden it's quite annoying in chat list.. is there an easy fix possibly?